kolibri/plugins/video_mp4_render/assets/src/vue/languagelookup.js

```diff
@@ -0,0 +1,925 @@
+/* eslint-disable */
+// Pulled from le-utils
+// https://github.com/learningequality/le-utils/blob/master/le_utils/resources/languagelookup.json
```

Comment:
Would be nice to require it, rather than copy it. Will look into whether we can do that in future, but this works for now!